charts/k8s-job/values.yaml

```diff
@@ -0,0 +1,164 @@
+#----------------------------------------------------------------------------------------------------------------------
```

Comment:
This is good to start! In the short term future right after this, we should also implement support for running [multiple copies of the pod with parallelism configuration](https://kubernetes.io/docs/concepts/workloads/controllers/job/#parallel-jobs).